src/Promenade.Web/Views/Podcasts/Episode.cshtml

```diff
@@ -9,10 +9,23 @@
                 @Localizer[Promenade.PodcastEpisodePublished, Model.PodcastItem.Episode, Model.PodcastItem.PublishDate.Value.ToLongDateString()]
             </em>
         </div>
-        <audio controls class="mb-2 w-100">
+        <audio id="player" controls class="mb-2 w-100">
             <source src="@Model.PodcastItem.MediaUrl" type="@Model.PodcastItem.MediaType" />
         </audio>
         <p>@Model.PodcastItem.Description</p>
         <div><em>@Localizer[Promenade.KeywordsItem, Model.PodcastItem.Keywords]</em></div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+@if (!string.IsNullOrWhiteSpace(Context.Items[Ocuda.Promenade.Controllers.ItemKey.GoogleAnalyticsTrackingCode] as string))
```

Comment:
Can we avoid this cast to string by either:

- Check if `Context.Items.ContainsKey()` (not sure if that method is available)
- Just check if `Context.Items[Key] == null`

Reply:
Resolved in commit 11b94af